docs/cases/cases-ui-integrations.asciidoc

```diff
@@ -24,138 +24,88 @@ https://www.elastic.co/subscriptions[appropriate license], and your role needs *
-you don’t set field mappings now, you’ll be prompted to do so if you want to use the connector for a case or a rule.
-
-| *Alerts* | Provide an alert ID and rule name.
-
-| *Cases* | Provide a case ID, a case name, comments, and a description.
-
-|===
+* *{sn} instance URL*: The URL of the {sn} instance to which you want to send cases.
+* *Use OAuth authentication*: Enable this to use open authorization (OAuth) to authenticate a connection between Elastic and {sn}.
 +
-. Save the connector.
+NOTE: To use open authorization (OAuth), you must {kibana-ref}/servicenow-action-type.html#servicenow-itsm-connector-prerequisites-rsa-key[create an RSA keypair and add an X.509 Certificate] and also {kibana-ref}/servicenow-action-type.html#servicenow-itsm-connector-prerequisites-endpoint[create an OAuth JWT API endpoint for external clients with a JWT Verifiers Map.]
++
+
+* *Username*: (Displays if *Use OAuth authentication* is turned _off_.) The username of the {sn} account used to access the {sn} instance.
+* *Password*: (Displays if *Use OAuth authentication* is turned _off_.) The password of the {sn} account used to access the {sn} instance.
+* *Client ID*: (Displays if *Use OAuth authentication* is turned _on_.) The client ID assigned to your OAuth application.
+* *User Identifier*: (Displays if *Use OAuth authentication* is turned _on_.) Identifier to use for OAuth type authentication. Use the value you entered into the *User field* when you created an OAuth JWT API endpoint for external clients.
+* *JWT Verifier Key ID*: (Displays if *Use OAuth authentication* is turned _on_.)
+The key ID assigned to the JWT Verifier Map of your OAuth application.
+* *Client Secret*: (Displays if *Use OAuth authentication* is turned _on_.) The client secret assigned to your OAuth application.
+* *Private Key*: (Displays if *Use OAuth authentication* is turned _on_.)
+The RSA private key generated when you created an RSA keypair.
+* *Private Key Password*: (Displays only if *Use OAuth authentication* is turned _on_.) The password for the RSA private key generated during setup, if set.
+
+** For {swimlane} connectors:
+
+* *API URL*: The URL of the {swimlane} instance to which you want to send cases.
+* *Application ID*: The application ID of your {swimlane} application. From {swimlane}, you can find the application
+ID by checking your application’s settings or at the end of your application’s URL after you’ve opened it.
+* *API token*: The {swimlane} API authentication token is used for HTTP Basic authentication.
+This is the personal access token for your user role.
+* *Connector Type*: Select a connector type:
+*** *All*: You can choose to set all or no field mappings when creating your new {swimlane} connector. However, note that if you don’t set field mappings now, you’ll be prompted to do so if you want to use the connector for a case or a rule.
```

Comment:
```suggestion
*** *All*: You can choose to set all or no field mappings when creating your new {swimlane} connector. However, note that if you don’t set field mappings now, you’ll be prompted to do so before you use the connector for a case or a rule.
```

Reply:
This is probably a dumb question because I've never tried out a Swimlane connector, but does this not apply to the other two options here ("Alerts" and "Cases")? I.e. do you not have to map fields if you select one of those? It reads like you don't, so just confirming.

Reply:
Not a dumb question and you're correct! I'll move this to a note under the parent bullet (so under **Connector Type**) because it does apply to all the types.